plugins/env/app/views/samson_env/_fields.html.erb

```diff
@@ -15,5 +15,6 @@
       </div>
     <% end %>
   <% end %>
+  <%= link_to "Add row", "#", class: "duplicate_previous_row" %> |
```

Comment:
sry for nitpicking.. is there a whitespace separating the pipe from the next link? Maybe use a html entity because lots of  people's text editor configs delete trailing whitespace?